syntax = "proto3";
package proto;

import "ByteMap.proto";

message ImageByteMap
{
    bytes uuid = 1;
    ByteMap byte_map = 2;
    string file_name = 3;
    uint64 ea_min = 4;
    uint64 ea_max = 5;
    uint64 base_address = 6;
    uint64 entry_point_address = 7;
    int64 rebase_delta = 8;
    bool is_relocated = 9;
}
